tidb-cloud/scale-tidb-cluster.md

```diff
@@ -80,7 +80,9 @@ You can increase or decrease the size (including vCPUs and memory) of TiDB, TiKV
 
 > **Note:**
 >
-> - Changing node size is only available to clusters that are hosted on AWS and created after 2022/12/31.
+> - Changing node size is only available to clusters that are:
+>     - hosted on AWS and created after 2022/12/31.
+>     - hosted on GCP and created after 2023/04/26.
```

Comment:
```suggestion
> - Changing node size is only available to the following clusters:
>     - hosted on AWS and created after 2022/12/31.
>     - hosted on GCP and created after 2023/04/26.
```

Reply:
When a colon introduces a list, the text that precedes the colon must be able to stand alone as a complete sentence.
See https://developers.google.cn/style/colons#introductory-phrase-preceding-colon